fitbenchmarking/results_processing/fitting_report.py

```diff
@@ -84,6 +84,8 @@ def create_prob_group(result, support_pages_dir, options):
             equation=result.problem.equation,
             initial_guess=result.ini_function_params,
             minimizer=result.minimizer,
+            accuracy=f"{result.chi_sq:.7f}",
+            runtime=f"{result.runtime:.7f}",
```

Comment:
I think g is generally cleaner as it uses significat figures rather than decimal places.
```suggestion
            accuracy=f"{result.chi_sq:.7g}",
            runtime=f"{result.runtime:.7g}",
```
Was there a reason for 7? It seems quite long but I guess we want to be informative.

Reply:
I chose 7 because that seemed to be the largest precision in the tables themselves.

Do you remember off the top of your head the number of significant figures used for the tables?

Reply:
Looks like 4 sig figs so I'll do that 👍